includes/js_defaults.inc.php

```diff
@@ -115,6 +115,15 @@
 settings["unmarked_image"] =               "<?php echo $theme_config['unmarked_image']; ?>";
 <?php endif; ?>
 
+var user_settings = new Array();
+<?php if (isset ($usersettings['browser_window_target']) and $usersettings['browser_window_target'] == 1): ?>
+user_settings["open_links_in_new_window"] = "EXTERNAL";
+<?php elseif (isset ($usersettings['browser_window_target']) and $usersettings['browser_window_target'] == 2): ?>
+user_settings["open_links_in_new_window"] = "ALL";
+<?php else: ?>
+user_settings["open_links_in_new_window"] = "NON";
```

Comment:
It looks fine, but if I review my own [code](https://github.com/ilosuna/mylittleforum/pull/546/files#diff-431e5adb210d9982cdf7bacdacad1bab1cd70c57028cb41316d8bb79d2d8b046R956), there are four cases, i.e., EXTERNAL, ALL, NONE, DEFAULT. The default-case uses the global forum settings, this is not none.

Reply:
I think, when I add the case `DEFAULT`, this has to be `0` in the database. So I have to shift all cases by 1. Any thoughts?

Reply:
Perfect solution.